structs.go

```diff
@@ -432,6 +432,12 @@ type GuildBan struct {
 	GuildID string `json:"guild_id"`
 }
 
+// A GuildBans stores data for a banned user.
+type GuildBans struct {
```

Comment:
Can't be GuildBan because GuildBan specify a guild however the GuildBans function already specified the guild and so doesn't have the guild argument (maybe can merge them if i put `GuildID string \`json:"guild_id"\`` has optionnal.